test/packaging/ansible/roles/repo-setup/tasks/repo-Ubuntu.yaml

```diff
@@ -12,6 +12,8 @@
     update_cache: no #ubuntu 20.10 image has broken repos. Cache will only be updated after NR repo is added
   loop: "{{ repos_to_clean }}"
 
+- debug: var=repo_endpoint
```

Comment:
leftover? ☝️

Reply:
I would leave it for some time to be sure the tests are running in the correct provided `repo_endpoint`.